Makefile

```diff
@@ -100,15 +96,20 @@ test-app/image_v1_signed.bin: test-app/image.bin
 
 factory.bin: $(BOOT_IMG) wolfboot-align.bin $(PRIVATE_KEY) test-app/image_v1_signed.bin
 	@echo "\t[MERGE] $@"
-	$(Q)cat wolfboot-align.bin test-app/image_v1_signed.bin > $@
+	$(Q)$(OBJCOPY) --set-start=$(ARCH_OFFSET) \
+		--add-section .signed_image=test-app/image_v1_signed.bin \
+		--change-section-vma .signed_image=$(WOLFBOOT_PARTITION_BOOT_ADDRESS) \
+		--set-section-flags .signed_image=alloc,readonly \
+		wolfboot.elf factory.elf
```

Comment:
Why do you call the output binary file "wolfboot.elf"? Is there a reason why you are using objcopy to create the factory image? This breaks the cortex-M build

Reply:
I abandoned this approach and have added bin-assemble to allow creating combined flash objects with some more flexibility. For T2080 the app image needs to be before the bootloader.